packages/dom/test/serialize-inputs.test.js

```diff
@@ -62,8 +62,10 @@ describe('serializeInputs', () => {
           option.selected = false;
         }
       });
-      cache[platform].dom = dom;
       cache[platform].$ = parseDOM(serializeDOM(), platform);
+      // save DOM, only after serializing it
+      // clone DOM, so that during next platform iteration, serializeDOM doesn't change current original DOM
+      cache[platform].dom = platform === 'shadow' ? dom : dom.cloneNode(true);
```

Comment:
Here is the test preparation workflow that is happening for Chrome -
1. We create `plain` DOM, and then run `serialzeDOM` on it, and store this in `cache['plain']`
2. Then we take the **same** DOM attach shadow components to it, and then run `serializeDOM` on it again, and store this in `cache['shadow']` 
3. So here, the `plain` DOM part is getting serialized twice, and since while storing in `cache['plain']` we store by reference, it get's updated to be the newly serialized plain DOM, whereas the serializedDOM from first iteration is intact.

Reply:
understood, in this case we shouldn't be calling `dom.cloneNode` rather our custom clone node traversal code, will update and merge.